src/noInputRenameRule.ts

```diff
@@ -90,17 +92,15 @@ export class InputMetadataWalker extends NgWalker {
       (propertyAlias !== propertyName &&
         this.directiveSelectors &&
         this.directiveSelectors.some(x => new RegExp(`^${x}((${toTitleCase(propertyName)}$)|(?=$))`).test(propertyAlias))) ||
-      (whiteListAliases.has(propertyAlias) && propertyName === kebabToCamelCase(propertyAlias))
+      (whiteListAliases.has(propertyAlias) && propertyName === camelize(propertyAlias))
```

Comment:
`kebabToCamelCase` is more semantically correct. We will not be able to `camelize` an arbitrary string. Would you mind reverting this?

Reply:
No problem. I did it because I saw that TSLint has already a function to camelize strings.